source/general/pkteam.rst

```diff
@@ -8,120 +8,140 @@ The ProjectKorra team is comprised of volunteers who work on various aspects of
 +-----------------+-----------------+---------------+
-| Jacklin213      | Developer       | |jacklin|     |
+| Jacklin213      | Developer       | |jacklin213|  |
 +-----------------+-----------------+---------------+
-| AlexTheCoder    | Developer       | |alex|        |
+| PhanaticD       | Developer       | |phanaticd|   |
 +-----------------+-----------------+---------------+
-| StrangeOne101   | Developer       | |strange|     |
+| plasmarob       | Developer       | |plasmarob|   |
 +-----------------+-----------------+---------------+
-| PhanaticD       | Developer       | |phanatic|    |
+| Simplicitee     | Developer       | |simp|        |
 +-----------------+-----------------+---------------+
-| Varhagna        | Developer       | |varhagna|    |
+| StrangeOne101   | Developer       ||strangeone101||
 +-----------------+-----------------+---------------+
-| Finn\_Bueno_    | Developer       | |finnbueno|   |
+
+Junior Developers
+^^^^^^^^^^^^^^^^^
+Junior Developers are developers in training. They help out where they can and work on expanding their skills to eventually become developers
+
 +-----------------+-----------------+---------------+
-| plasmarob       | Developer       | |plasmarob|   |
+| Varhagna        | Developer       | |varhagna|    |
++-----------------+-----------------+---------------+
+
+Mod Developers
+^^^^^^^^^^^^^^
+Mod Developers work on the modded side of ProjectKorra. They are responsible for maintaining Avatar Mod 2: Out of the Iceberg, which you can find here.
+
 +-----------------+-----------------+---------------+
 | FavouriteDragon | Forge Developer | |favourite|   |
 +-----------------+-----------------+---------------+
 | Mahtaran        | Forge Developer | |mahtaran|    |
 +-----------------+-----------------+---------------+
 | Mnesikos        | Forge Developer | |mnesikos|    |
++-----------------+-----------------+---------------+
+
+Bug Testers
+^^^^^^^^^^^
+Bug testers work on quality assurance. They are the ones who vigourously test ProjectKorra plugins to make sure they are usuable.
+
 +-----------------+-----------------+---------------+
 | TheWaterChief   | Bug Tester      | |waterchief|  |
 +-----------------+-----------------+---------------+
 
-Moderators
-----------
 
-Moderators work on the forums and discord to keep them
-organized, civil, and spam-free.
+Digital Team
+------------
+The Digital Team focuses making ProjectKorra look good. Their main responsibilities include making videos, maintaining our wiki, and managing social media. 
+
+If you want to join the digital team, click the link below to apply
+
+`Join the Digital Team! <https://projectkorra.com/join-the-team/>`_
+
+Wiki Maintainers
+^^^^^^^^^^^^^^^^
 
 +-----------------+-----------------+---------------+
-| Pickle9775      | Head Moderator  | |pickle|      |
-+-----------------+-----------------+---------------+
-| floory565       | Moderator       | |floory|      |
+| Jacksonnn       | Wiki Editor     | |jackson|     |
```

Comment:
Wiki Editor is not my only role as Digital Team, it would be more accurate for me to have Digital Team as my title instead of just wiki editor.

This is my thought at least.

Reply:
true